asistente_ladm_col/lib/dbconnector/pg_connector.py

```diff
@@ -47,6 +47,20 @@ def _schema_exists(self):
 
         return False
 
+    def _postgis_exists(self):
+        if self.schema:
```

Comment:
We don't need to check for schemas here, it is the database that has or does not have PostGIS installed.

Reply:
Fixed in https://github.com/AgenciaImplementacion/Asistente-LADM_COL/commit/11bcc5ceee4ae56ed1a60855067fcd9d9c7a8589